.github/release-please.yml

```diff
@@ -1,6 +1,7 @@
 primaryBranch: main
 releaseType: java-yoshi
 handleGHRelease: true
+last-release-sha: 103846dbe135790a59788a5ae3bafe56ededba3e
```

Comment:
This won't work in the app's config. It has to be in `release-please-config.json`. The other config options would need to port to that file too.

Then set `manifest: true` in this app config to tell it to use the `release-please-config.json`. We highly recommend using that config file because then you can also test your settings using the `release-please` CLI as well

Reply:
Okay - makes sense. Thank you for the clarification.

After discussing with @suztomo , we've reset the v5.0.0 release to a commit on the main branch, so this should allow release please to correct itself with the current configuration.